WordPress/src/main/java/org/wordpress/android/util/AnalyticsUtils.java

```diff
@@ -65,7 +65,7 @@ public static void refreshMetadata(AccountStore accountStore, SiteStore siteStor
      * counts Jetpack sites connected via WPCom Rest API. If there are Jetpack sites in the site store and they're
      * all accessed via XMLRPC, this method returns false.
      */
-    private static boolean isJetpackUser(SiteStore siteStore) {
+    public static boolean isJetpackUser(SiteStore siteStore) {
```

Comment:
I'd remove `public` here and just use `static boolean isJetpackUser(SiteStore siteStore)` instead since it's package-private. This way AS can stop bugging us about it. However, that's not something we do often, so it doesn't matter much.